Source/WebCore/page/scrolling/ScrollingStateTree.h

```diff
@@ -111,6 +113,8 @@ class ScrollingStateTree : public CanMakeCheckedPtr {
     void traverse(const ScrollingStateNode&, const Function<void(const ScrollingStateNode&)>&) const;
 
     ThreadSafeWeakPtr<AsyncScrollingCoordinator> m_scrollingCoordinator;
+    FrameIdentifier m_frameIdentifier;
```

Comment:
Maybe m_rootFrameIdentifier or whatever we call the root-most frame in an isolated process.

Reply:
Fixed.